build.gradle

```diff
@@ -81,6 +81,7 @@ dependencies {
     runtimeOnly("javax.xml.bind:jaxb-api:2.3.1")
     testImplementation("org.testcontainers:testcontainers")
     testImplementation("org.testcontainers:mysql:1.17.3")
+    testImplementation 'com.github.tomakehurst:wiremock:3.0.1'
```

Comment:
Likely this is not needed, because there isn't the interaction with the License manager anymore

Reply:
removed